### PR TITLE
Feature/137 update template

### PR DESCRIPTION
**Pull request by maxdiebold-erg**
## Related Issues:
* HMW-137

## Main Changes:
* Updated the HTML templates to match those on the main [https://www.epa.gov]() site.
* The EPA navigation links above the HMW content are now drop-down menus.

## Steps To Test:
1. Go to [http://localhost:3000/](), and browse any site under the `mywaterway` subdomain
2. Decrease the screen width to confirm responsiveness on mobile browsers
3. Disable JavaScript to confirm that the interface adapts and the proper alerts are presented to the user

## TODO:
- [ ] I included the "return to top" button from [https://www.epa.gov](), but it might get in the way of the HMW application, so it could use some usability testing.

### CODE REVIEW

**Comment by maxdiebold-erg** [comment]
@cschwinderg

**Comment by maxdiebold-erg** [comment]
@cschwinderg yeah it broke a lot more than I thought, I'll work on that now.

File: app/client/public/index.html [hunk 1]
```diff
@@ -77,30 +79,208 @@
...
+        max-width: 60.24rem;
+        padding-left: 1.882rem;
+        padding-right: 1.882rem;
       }
 
-      .main-footer .region-footer .col:last-of-type {
-        padding-right: 0;
+      .l-page {
+        padding-top: 0rem;
+      }
```
Comment (cschwinderg) [inline app/client/public/index.html:255]:
```suggestion
```

Related to the comment above, this one should be deleted.

**Review by cschwinderg — CHANGES_REQUESTED** [review]
@maxdiebold-erg I'm not sure if this is complete yet, but I figured I would take a look while I was fixing the merge conflicts. 

This looks good! Thanks for working on this.

Can you fix the minor code smell that sonar cloud flagged? The comments below are related to that code smell.

**Comment by maxdiebold-erg** [comment]
Yes, I'll take care of that. I'm not positive it's complete either, but as far as I can see things fit!

**Pull request closed without merging**